.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+To help us get this pull request reviewed and merged quickly, please be sure to include the following items:
+```
+[ ] Tests (if applicable)
+[ ] Documentation (if applicable)
+[ ] Changelog entry
+[ ] A full explanation here in the PR description of the work done
+```
```

Comment:
```suggestion

* [ ] Tests (if applicable)
* [ ] Documentation (if applicable)
* [ ] Changelog entry
* [ ] A full explanation here in the PR description of the work done

```
If you leave them out of the code fencing and make them a list, GitHub should actually turn these into checkboxes for the user to check. :)